src/pages/Browse/index.spec.tsx

```diff
@@ -181,4 +353,57 @@ describe(__filename, () => {
       path,
     });
   });
+
+  it('renders a global LinterMessage', () => {
+    const file = 'lib/react.js';
+    const externalMessage = globalExternalMessage({
+      file,
+      message: 'This is a known third party library',
```

Comment:
This doesn't seem strictly needed, as we're never asserting anything about it. Could this not just be the default message that comes with `fakeExternalLinterMessage`?